workspace/Cargo.toml

```diff
@@ -27,7 +28,7 @@ lazy_static = "1.4"
 tokio = { version = "1" }
 
 [features]
-default = []
+default = ["mock"]
```

Comment:
Why was this enabled? Just making sure it's not a temporary change.

Reply:
This feature means we send arguments to the `new` function in json format. Without it, it would be in bosh. I don't get the point of this, and therefore just remove it. The mock contract waits for it in json, but FYI, aurora EVM waits for these args in bosh. Probably we should make it determinate. So the mock smart contract should have the same methods aurora EVM has with identical signatures and type of params serialising.

Reply:
Yes, that's a good point.